pkg/metadata/host/host_notwin.go

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
we should rename the file `host_nix.go` to follow what is done elsewhere in the agent.

Reply:
There was already a `host_nix.go` with different build flags.

Reply:
indeed !